src/main/resources/termsOfService.md

```diff
@@ -160,20 +162,20 @@ legitimate tasks or assigned duties. If you believe you are being
 granted access that you should not have, you must immediately notify the
 Broad Operations Center at infosec@broadinstitute.org.
 
-## Restricted Use of TCGA Controlled-Access Data
+## Restricted Use of Federal Controlled-Access Data
 
-To access TCGA controlled access data, you must first request to link
+To access federal controlled access data, you must first request to link
 your Terra account to the eRA Commons or NIH identity under which you
 are granted dbGaP authorized access. If you have dbGaP authorized
-access, Terra permits read access to the TCGA controlled-access data
-files.
+access, Terra permits read access to those the federal controlled-access
+data files.
 
 If the NIH or eRA commons identity authentication fails or the
 authenticated identity no longer appears in the NIH-provided whitelist,
-you can no longer access TCGA controlled access data files. It is your
-responsibility to delete any TCGA controlled-access data in your
+you can no longer access federal controlled access data files. It is your
```

Comment:
Should be "controlled-access data files" for consistency with the other usages.

Reply:
I think we should change this one given the usage in the very next sentence…

Reply:
I agree that it's inconsistent, but for context, this is what the legal team provided to us verbatim. I personally don't feel comfortable making even minor adjustments to a legal document that's been reviewed and agreed upon by the Broad legal team and the BVM partners.